### PR TITLE
1376 export column labels

### PR DESCRIPTION
**Pull request by willdoran**
This pull request makes the following changes:
- Changing Posts Export to its own usecase and updating column naming to use form attribute label rather than key

Test these changes by:
-

Fixes ushahidi/platform#1376

Ping @ushahidi/platform

---
This change is [Reviewable](https://reviewable.io/reviews/ushahidi/platform/1415)

### CODE REVIEW

File: application/classes/Ushahidi/Repository/Post/Export.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+use Ushahidi\Core\Entity\Post;
+use Ushahidi\Core\Entity\PostRepository;
+
+class Ushahidi_Repository_Post_Export extends Ushahidi_Repository_Post
+{
+  public function getFormAttributes($values) {
+    $attributes = [];
+		foreach ($values as $key => $val)
```
Comment (jasonmule) [inline application/classes/Ushahidi/Repository/Post/Export.php]: Might need to fix indentation?

File: src/Core/Usecase/Post/ExportPost.php [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+ * @package    Ushahidi\Platform
+ * @copyright  2014 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+namespace Ushahidi\Core\Usecase\Post;
+
+use Ushahidi\Core\Usecase\SearchUsecase;
+
+class ExportPost extends SearchUsecase
```
Comment (jasonmule) [inline src/Core/Usecase/Post/ExportPost.php]: Maybe just `Export`

Comment (zhalloran) [inline src/Core/Usecase/Post/ExportPost.php]: Agreed. Given the namespace Export is sufficient for the class name.

File: application/classes/Ushahidi/Formatter/Post/CSV.php [hunk 3]
```diff
@@ -152,16 +153,21 @@ protected function getCSVHeading($records)
...
 			foreach ($record as $key => $val)
 			{
 				// Assign form keys
 				if ($key == 'values')
 				{
+
 					foreach ($val as $key => $val)
 					{
-						$this->assignColumnHeading($columns, $key, $val[0]);
+						$label = $attributes[$key].'.'.$key;
```
Comment (zhalloran) [inline application/classes/Ushahidi/Formatter/Post/CSV.php]: Error handling for a missing `$attributes[$key]`

File: application/classes/Ushahidi/Repository/Post/Export.php (hunk 1 above)
Comment (willdoran) [inline application/classes/Ushahidi/Repository/Post/Export.php]: done

File: src/Core/Usecase/Post/ExportPost.php (hunk 2 above)
Comment (willdoran) [inline src/Core/Usecase/Post/ExportPost.php]: done

**Pull request merged**